purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/offlineentitlements/OfflineEntitlementsWithInitialRequestsCompletedIntegrationTest.kt

```diff
@@ -299,11 +322,12 @@ class OfflineEntitlementsWithInitialRequestsCompletedAndNoInitialPurchasesIntegr
                         onError = {
                             fail("Expected success but got error: $it")
                         },
-                        onSuccess = {
-                            // This is a known limitation. Ideally we would sync unsynced purchases
-                            // as soon as possible to avoid getting outdated info from the backend.
-                            assertCustomerInfoDoesNotHavePurchaseData(it)
-                            assertAcknowledgePurchaseDidNotHappen()
+                        onSuccess = { customerInfo2 ->
+                            // This is because the token we are using is not a real token to be used in the backend
```

Comment:
We should have an active token so we can test active entitlements more easily in production. Right now, production will not return active purchases since the token is expired.